### ISSUE TITLE
Error with babel/register

### ISSUE THREAD

**Issue by gsklee** [babel/babel#1689]
Upgraded from 5.4.7 to 5.5.1 and am having this error:

```
> webpack-dev-server --config ./babel-register.js --inline --hot

/srv/home/kaylee/projects-x/node_modules/webpack/lib/HotModuleReplacement.runtime.js:38
			0++;
			^
ReferenceError: Invalid left-hand side expression in postfix operation
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:409:25)
    at normalLoader (/srv/home/kaylee/projects-x/node_modules/babel-core/lib/babel/api/register/node.js:160:5)
    at Object.require.extensions.(anonymous function) [as .js] (/srv/home/kaylee/projects-x/node_modules/babel-core/lib/babel/api/register/node.js:173:7)
    at Module.load (module.js:351:32)
    at Function.Module._load (module.js:306:12)
    at Module.require (module.js:361:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/srv/home/kaylee/projects-x/node_modules/webpack/lib/HotModuleReplacementPlugin.js:222:47)
    at Module._compile (module.js:426:26)
```

**babel-register.js**
```javascript
require('babel/register');

module.exports = require(({
  'webpack-dev-server': './webpack.config',
  'gulp': './gulpfile'
})[process.argv[1].split('/').pop()]);
```

**Comment by sebmck** [comment]
Turn off the `minification` transformers. Also I need the actual code that produces the error, I literally cannot debug further with so little information.

**Comment by gsklee** [comment]
This is my webpack.config.js:

```javascript
import {assign, keys} from 'bound-native-methods/object';
import OS from 'os';
import Path from 'path';
import Webpack from 'webpack';

const isProduction = process.env.NODE_ENV === 'production',

      interfaces = OS.networkInterfaces(),

      privateInterface = [].concat(...interfaces::keys().map(name => interfaces[name]))
                           .find(({address, family, internal}) =>
                             !internal && family === 'IPv4' &&
                               address.split('.', 2).toString() === '192,168'
                           );

export default {
  entry: './source/scripts/main.js',

  output: {
    filename: 'main.js'
  },

  module: {
    loaders: [{
      test: /\.css$/,
      loader: 'style!css'
    }, {
      test: /\.js$/,
      exclude: /node_modules/,
      loader: 'react-hot!babel'
    }]
  },

  plugins: [
    new Webpack.optimize.DedupePlugin(),
    new Webpack.optimize.OccurenceOrderPlugin(true)
  ]
}::assign(!isProduction && {
  cache: true,
  debug: true,
  devtool: 'eval',

  devServer: {
    host: privateInterface.address || '0.0.0.0',
    port: 9090,
    contentBase: './source',
    colors: true
  }
});
```

**Comment by sebmck** [comment]
Due to a regression introduced in #1665. Fixed as of 5.5.2, thanks!

**Comment by gsklee** [comment]
@sebmck Cool :+1:
